iib/workers/tasks/utils.py

```diff
@@ -25,6 +28,35 @@ def get_image_labels(pull_spec):
     return skopeo_inspect(full_pull_spec).get('Labels', {})
 
 
+def retry(
+    retries=worker_config.worker_total_retries, wait_on=Exception, logger=None,
```

Comment:
To avoid a global variable, you can do the following. `get_worker_config` will be executed only once when the module is imported this approach.

```suggestion
    retries=get_worker_config().worker_total_retries, wait_on=Exception, logger=None,
```